dd-java-agent/instrumentation/gradle-testing/build.gradle

```
@@ -0,0 +1,17 @@
+apply from: "$rootDir/gradle/java.gradle"
+
+repositories {
+  maven {
+    url "https://repo.gradle.org/artifactory/libs-releases-local"
+  }
+}
+
+// gradle >= v5.1
+dependencies {
+  compileOnly gradleApi()
+  compileOnly project(":dd-java-agent:instrumentation:junit-4.10")
```

Even though it makes no difference now (as we have no muzzle and no instrumentation tests for this module):
```suggestion
  implementation project(":dd-java-agent:instrumentation:junit-4.10")
```